programs/ssl/ssl_client2.c

```diff
@@ -901,6 +903,16 @@ int main( int argc, char *argv[] )
             if( opt.debug_level < 0 || opt.debug_level > 65535 )
                 goto usage;
         }
+        else if( strcmp( p, "build_version" ) == 0 )
+        {
+            if( strcmp( q, "1" ) == 0 )
+            {
+                mbedtls_printf( "build version: %s (build %d)\n",
+                                MBEDTLS_VERSION_STRING_FULL,
```

Comment:
In `development`, `mbedtls/version.h` is getting included indirectly in via `mbedtls/build_info.h`, but this doesn't happen in 2.28. Please add `#include "mbedtls/version.h"` to `ssl_test_lib.h`